src/languages/julia.js

```diff
@@ -84,7 +84,7 @@ function(hljs) {
 
   // placeholder for recursive self-reference
   var DEFAULT = {
-    aliases: ['jldoctest'],
+    aliases: ['julia.console'],
```

Comment:
@isagalaev, how would one actually use an alias with a period in it? It seems that doing e.g. `class="language-julia.console` actually just picks `julia`, ignoring everything after the period (e.g. if I create an alias `juliax.console` I can't select it at all).